app/src/test/java/com/hover/stax/utils/UtilsTest.kt

```diff
@@ -0,0 +1,13 @@
+package com.hover.stax.utils
+
+import org.junit.Assert
+import org.junit.Test
+
+
+class UtilsTest {
+	@Test
```

Comment:
Should have a couple of tests to check what happens with eg `43`, `43.1` and `43.986` since it is possible to get any of these back from various USSD services.

Reply:
Ideally the different inputs should be separate tests so that you can name them and anyone can know what you're testing  for